builds/misc/images-release.yaml

```diff
@@ -25,8 +25,6 @@ jobs:
         inputs:
           command: login
           containerRegistry: iotedge-release-acr
-      # Dotnet 2 needed for codesign
```

Comment:
Oops, I made a mistake. We still need .NET Core 2.x for Codesign, as part of the release process. So:
1. `templates/install-dotnet2.yaml` should _not_ be removed (again, sorry 😔)
2. The references to `install-dotnet2.yaml` in `images-release.yaml` should _not_ be removed
3. References to `install-dotnet2.yaml` in all other YAML files _should_ be removed

Reply:
for linux it uses ../templates/dotnet2-globaljson.yaml # switch to dotnet 2 as primary install for code sign, I think I can use that, any conserns?

Reply:
`dotnet2-globaljson.yaml` doesn't _install_ .NET Core 2.x, it only _configures_ .NET Core to use 2.x by default. I think you need to put back `templates/install-dotnet2.yaml` (in this file only) to ensure 2.x is installed.

Reply:
thanks! it makes sense, I added dotnet2 back. While doing this I noticed that metrics-collector-images-release.yaml doesn't sign, it is signed in images-release. It looks like sign should be added to metrics-collector-images-release.yaml

Reply:
lol...was _just_ talking to @and-rewsmith and @yophilav about the same thing! We'll take care of that...